Solutions/Recorded Future/Data/Solution_RecordedFuture.json

```diff
@@ -25,7 +25,7 @@
     "Workbooks/Recorded Future - Actively Communicating C&C IPs to DNS Events - Correlation&Threat Hunting.json"
   ],
   "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\Recorded Future",
-  "Version": "2.2.2",
+  "Version": "2.3.0",
```

Comment:
@RecordedFutureOskbo, can you please check the version for solution, should it be 2.2.3?

Reply:
We communicated this release as 2.3 internally due to the added detections.